### ISSUE TITLE
"class" specifier

### ISSUE THREAD

**Issue by davidebeatrici** [Yimura/GTAV-Classes#5]
Right now its usage is inconsistent:

https://github.com/Yimura/GTAV-Classes/blob/48bd9e37dcdba89d0249917f9684ee46a2aad697/CNavigation.hpp#L8-L14

https://github.com/Yimura/GTAV-Classes/blob/48bd9e37dcdba89d0249917f9684ee46a2aad697/fwEntity.hpp#L14-L32

I would personally remove it everywhere.

**Comment by davidebeatrici** [comment]
Specifying `class` for `rage::fvector3` is actually causing a compiler error right now:

```cpp
fwEntity.hpp:29:21: error: typedef 'fvector3' cannot be referenced with a class specifier
        class rage::fvector3 m_right //0x0060
                    ^
vector.hpp:12:28: note: declared here
    typedef vector3<float> fvector3
                           ^
```

**Comment by Yimura** [comment]
Do you remember you introduced this inconsistency through your PR?

I never checked if this was cimpilable or not, either make a PR fixing the issue or wait when I make time to fix the issue.

**Comment by Yimura** [comment]
Additionally the class specifier is automatically added by ReClass, I never bothered removing it.

**Comment by davidebeatrici** [comment]
> Do you remember you introduced this inconsistency through your PR?

You're right, sorry about that. I got confused by the other class members and didn't realize `rage::fvector3` was not a class.

> Additionally the class specifier is automatically added by ReClass, I never bothered removing it.

I see, no problem.
